src/pages/dev/general-message-passing/examples.mdx

```diff
@@ -1,9 +1,22 @@
-#  Example cross-chain dApps using GMP
+import { Callout } from "/src/components/callout";
+
+# Example cross-chain dApps using GMP
 
 The [axelar-examples](https://github.com/axelarnetwork/axelar-examples/) repo has an array of examples to show the ease of integrating Axelar's GMP protocol into any dApp to bring it cross-chain. The examples range in use cases and complexity but ultimately leverage the two fundamental building blocks for GMP:
+
 - `callContract`
 - `callContractWithToken`
 
+<Callout emoji="ℹ️">
+  NOTE: Due to recent Solidity language upgrades, the
+  <a href="https://www.zaryabs.com/push0-opcode/">`PUSH0`</a> opcode is now generated
```

Comment:
This is only an issue when using the new default EVM version of shanghai when compiling. Instead, devs can set the version to an older one such as `london` for compatibility on another chains.

Reply:
Ok good callout, I'll add that as an extra note on the doc and leave it up to the devs if they want to rollback their Solidity version or their EVM version